### PR TITLE
[Bug 22443] PB align widget "icon" with connectors

### PR DESCRIPTION
**Pull request by BerndN**

### CODE REVIEW

File: notes/bugfix-22443.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# # PB align widget "icon" with connectors
```
Comment (livecodepanos) [inline notes/bugfix-22443.md]: This has an extra `#` in the beginning. Also I suggest modifying the note to:

`# Align widget "icon" with connectors in Project Browser`

Comment (BerndN) [reply notes/bugfix-22443.md]: Thanks Panos, done

**Comment by livecodepanos** [comment]
Thank you @BerndN :)

@livecode-vulcan review ok b67f1ab

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok b67f1ab4f292e7d59b2fdd73d759f7b64da4d248

**Pull request merged**